docker/jenkins/build.sh

```diff
@@ -23,14 +23,14 @@ then
 elif [ "${BUILD_TYPE}" == "gcc71" ]
 then
     source ../scripts/docker_gcc71_env.sh
-    ../scripts/docker_cmake -D Trilinos_CXX11_FLAGS="-std=c++17" -D Trilinos_ENABLE_OpenMP=OFF -D Tpetra_INST_OPENMP=OFF -D Kokkos_ENABLE_OpenMP=OFF
+    ../scripts/docker_cmake -D Trilinos_CXX11_FLAGS="-std=c++17" -D Trilinos_ENABLE_OpenMP=OFF -D Tpetra_INST_OPENMP=OFF -D Kokkos_ENABLE_OpenMP=OFF -D Trilinos_ENABLE_Fortran=ON
```

Comment:
Is the default for `Trilinos_ENABLE_Fortran` actually `OFF` or is it just not defined?

Reply:
Very good point. Just tried removing it and it still builds fine.